apollo-normalized-cache-api/src/commonMain/kotlin/com/apollographql/apollo/cache/normalized/Record.kt

```diff
@@ -1,141 +1,65 @@
 package com.apollographql.apollo.cache.normalized
 
-import com.apollographql.apollo.cache.normalized.internal.RecordWeigher.byteChange
 import com.apollographql.apollo.cache.normalized.internal.RecordWeigher.calculateBytes
 import com.benasher44.uuid.Uuid
-import kotlin.jvm.JvmStatic
-import kotlin.jvm.Synchronized
-import kotlin.jvm.Volatile
 
 /**
  * A normalized entry that corresponds to a response object. Object fields are stored if they are a GraphQL Scalars. If
  * a field is a GraphQL Object a [CacheReference] will be stored instead.
  */
-class Record internal constructor(
+data class Record (
```

Comment:
Same question I always ask, sorry in advance 😅: why do we need the `data` there? It looks like we only need a `copy` that we can implement easily. That'll remove a bunch of bytecode and make evolving the API easier, cf https://jakewharton.com/public-api-challenges-in-kotlin/

Reply:
First I thought we needed `copy` function but we don't.